api/handlers/task_handler.go

```diff
@@ -119,13 +122,13 @@ func (this *TaskHandler) CreateTask(request *restful.Request, response *restful.
 		return
 	}
 
-	task, err := this.TaskLogic.CreateTask(req)
+	job, err := this.JobLogic.CreateJob(types.CreateTaskJob, req)
```

Comment:
Major change is that we're creating a job executed by the `runner` instead of executing the task right then and there